docs/manuals/core/config/data-node-config.md

```diff
@@ -41,6 +41,10 @@ The attributes available on data nodes are:
     storage type. It is copied in the dictionary properties of all the data nodes
     instantiated from this data node configuration.<br/>
 
+!!! Warning
+
+    Note that we cannot use the word "_entity_owner" as a key in the properties as it has been reserved for internal use.
```

Comment:
No problem with the underscore while parsing the Markdown? No italics?

Reply:
not at all!

Reply:
👌